VerificationErrors.dfy(8,38): Error: could not prove assertion
VerificationErrors.dfy(15,18): Error: could not prove assertion

Dafny program verifier finished with 0 verified, 2 errors
